content/docs/tasks/traffic-management/custom-gateway/index.md

```diff
@@ -0,0 +1,372 @@
+---
+title: Add a new Custom Ingress Gateway
+description: Describes how to add a new custom ingress gateway manually.
+weight: 89
+keywords: [ingress,traffic-management]
+---
+
+This post provides instructions to manually create a new custom ingress [gateway](/docs/reference/config/istio.networking.v1alpha3/#Gateway) with automatic provisioning of certificates based on cert-manager.
+This task was validated on AWS Infrastructure.
+
+## Before you begin
+
+* Setup Istio by following the instructions in the
+  [Installation guide](/docs/setup/).
+
+## Configuring the custom ingress gateway
+
+1. Check if [cert-manager](https://github.com/helm/charts/tree/master/stable/cert-manager) was installed using Helm with the following command:
+
+    {{< text bash >}}
+
+    $ helm ls
+
+    {{< /text>}}
+
+    The output should be similar to the example below and show cert-manager with a `STATUS` of `DEPLOYED`:
+
+    {{< text plaintext >}}
+
+    NAME   REVISION UPDATED                  STATUS   CHART                     APP VERSION   NAMESPACE
+    istio     1     Thu Oct 11 13:34:24 2018 DEPLOYED istio-1.0.2               1.0.2         istio-system
+    cert      1     Wed Oct 24 14:08:36 2018 DEPLOYED cert-manager-v0.6.0-dev.2 v0.6.0-dev.2  istio-system
+
+    {{< /text >}}
```

Comment:
What if it is not installed? What are the nstructions for running it?

Reply:
Updated with link to chart installation